pkg/state/temp.go

```diff
@@ -28,18 +28,26 @@ func createTempValuesFile(release *ReleaseSpec, data interface{}) (*os.File, err
 	return f, nil
 }
 
-func tempValuesFilePath(release *ReleaseSpec, data interface{}) (*string, error) {
-	id, err := generateValuesID(release, data)
-	if err != nil {
-		return nil, err
-	}
-
+func mkdirTemp() (string, error) {
```

Comment:
Would you mind separating refactoring PRs vs PRs that fix something, so that it's easier to review and reverted if it turned out to break anything?